src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.File.Create.cs

```diff
@@ -27,40 +27,39 @@ public void FileSystemWatcher_File_Create()
             }
         }
 
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "#34017")]
-        [OuterLoop]
         [Fact]
-        public void FileSystemWatcher_File_Create_MultipleWatchers_ExecutionContextFlowed()
+        public void FileSystemWatcher_File_Create_EnablingDisablingNotAffectRaisingEvent()
```

Comment:
Could this use the ExpectEvent helper?  Or if not, use ExecuteWithRetry around the whole body?  We've had too many CI failures due to FSW flakiness, so we have retries in pretty much all of our FSW tests that actually expect events.  The ExpectEvent helper has a built-in retry.

Reply:
wrapped in ExecuteWithRetry helper method